docs/source/dependencies.rst

```diff
@@ -38,7 +38,7 @@ Where ``<packagename>`` is one of the following:
 +-------------------------+---------------------------------------------------------------------+
 | Feature                 | Python package(s) to install                                        |
 +=========================+=====================================================================+
-| Pushover reporter       | `chump <https://github.com/karanlyons/chump/>`__                    |
+| Pushover reporter       | `chump <https://github.com/karanlyons/chump>`__                     |
```

Comment:
Unrelated change to this PR, can you move it into a separate PR (yeah, it's a one-character change, but just to keep it separate).

Reply:
Fair enough!